frontend/src/metabase/components/Modal.jsx

```diff
@@ -100,26 +100,30 @@ export class WindowModal extends Component {
         container={this._modalElement}
         enableMouseEvents={enableMouseEvents}
       >
-        <CSSTransitionGroup
-          component="div"
```

Comment:
in v1 the default component is `span` but in v4 the default is `div` so there's no need to specify this anymore.